src/core/services/theming/theming.js

```diff
@@ -242,6 +270,53 @@ function ThemingProvider($mdColorPalette) {
 
   // Default theme defined in core.js
 
+  /**
+   *
+   * @param color
+   * @returns {Function}
+   */
+  var setBrowserColor = function (color) {
+    <!-- Chrome, Firefox OS and Opera -->
```

Comment:
Not sure why you use ta HTML comment-opening sequence here, would be more consistent to use a basic `//` or `/* */`